### ISSUE TITLE
Integrate the applet inside system tray?

### ISSUE THREAD

**Issue by FedeDP** [kotelnik/plasma-applet-weather-widget#3]
Hi!

I really like your applet! But i cannot stop myself from wondering if integrating that in the system tray wouldn't be better still. I mean, it would be definitely an improvement.
Would this be hard to implement?

Thanks for your work!

**Comment by kotelnik** [comment]
Hi! I could probably try to do that. But still I wonder how the meteogram picture will look like in the much smaller unfolded notification area. I'll investigate this a little bit and try to think of something. Now I'm on holiday so it will be no sooner then the second half of July.

**Comment by FedeDP** [comment]
Yes, i thought about that too. What about giving only 3-days forecast without image, only temperature for different hours, as you do in the lower part of the widget?

By the way, thank you very much and ...have a nice holiday :D

**Comment by kotelnik** [comment]
Yes, this can look nice. Thanks for suggestion. And thanks, I will :))

**Comment by kotelnik** [comment]
Hi! So first version with systray support is here. I already know that the expanded plasmoid in systray doesn't look nice. I'll do something about it later. So I'm leaving this issue open if somebody would like to propose a visual or until I come up with nicer look.

**Comment by FedeDP** [comment]
You did a really nice work indeed :)
Thank you very much, now it looks much cleaner and visually integrated!

**Comment by FedeDP** [comment]
I have another proposal: if "automatic" (showing) is set in system tray settings, under "elements", hide the widget after a timeout of may be 10minutes after a refresh (manual or automatic), and show it again for the same amount of time, when it gets refreshed.

**Comment by kotelnik** [comment]
Thanks for suggestion! Check out the latest commit - I've added an option for that. Though I'm thinking about resetting the shown-in-tray-timeout after temperature/icon really changed instead of refreshed event. What do you think?

**Comment by FedeDP** [comment]
Yes, you're idea is better still!
Thanks for the really quick response :)

**Comment by ShalokShalom** [comment]
Hi there, where can i find the Systray icon(s) please?

I mean, how is the path to it?

**Comment by kotelnik** [comment]
Ops, sorry for the super late response.
Here: /usr/share/plasma/plasmoids/org.kde.weatherWidget/contents/images/weather-widget.svg
or here: ~/.local/share/plasma/plasmoids/org.kde.weatherWidget/contents/images/weather-widget.svg
depending on how you've installed the widget.
